pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTVariableDeclarationStatements.java

```diff
@@ -4,15 +4,18 @@
 
 package net.sourceforge.pmd.lang.apex.ast;
 
-import com.google.summit.ast.Node;
+import java.util.List;
+
 import net.sourceforge.pmd.annotation.InternalApi;
 
-public class ASTVariableDeclarationStatements extends AbstractApexNode.Single<Node> {
+import com.google.summit.ast.declaration.VariableDeclaration;
+
+public class ASTVariableDeclarationStatements extends AbstractApexNode.Many<VariableDeclaration> {
 
     @Deprecated
     @InternalApi
```

Comment:
Please remove these annotations and make the constructor package private

Reply:
Removed.

I noted a task to make a pass through all of the classes.